supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -38,6 +36,29 @@ When it is not possible to remove the terms 'blacklist' and 'whitelist', replace
 * denylist / allowlist
 * blocklist / passlist
 
+=== master and slave
+
+.Example: Removing 'master'
+image:images/no.png[no] A Ceph Monitor maintains the master copy of the Red Hat Ceph Storage cluster map with the current state of the Red Hat Ceph Storage cluster.
+
+image:images/yes.png[yes] A Ceph Monitor maintains the primary copy of the Red Hat Ceph Storage cluster map with the current state of the Red Hat Ceph Storage cluster.
+
+image:images/yes.png[yes] A Ceph Monitor maintains the main copy of the Red Hat Ceph Storage cluster map with the current state of the Red Hat Ceph Storage cluster.
+
+
+.Example: Removing 'slave'
+image:images/no.png[no] Copy the public key to the slave node using the following command.
+
+image:images/no.png[yes] Copy the public key to the secondary node using the following command.
+
+You can use the following alternatives:
```

Comment:
Move this section to precede the examples. Right now, it scans that this section is part of the slave example.